integration-tests/cli/personal-data-erasure.it.js

```diff
@@ -89,63 +90,77 @@ describe('personal data erasure', () => {
           { ...review[0], customer: { id: customerId, typeId: 'customer' } },
         ]),
       ])
-    })
-
-    afterAll(async () => {
-      await Promise.all([
-        clearData(apiConfig, 'customers'),
-        clearData(apiConfig, 'orders'),
-        clearData(apiConfig, 'carts'),
-        clearData(apiConfig, 'payments'),
-        clearData(apiConfig, 'shoppingLists'),
-        clearData(apiConfig, 'reviews'),
-      ])
-    })
+    }
 
     beforeEach(() => {
       personalDataErasure = new PersonalDataErasure({ apiConfig }, logger)
     })
 
-    describe('export function', () => {
-      it('should get data on the CTP', async () => {
-        const data = await personalDataErasure.getCustomerData(customerId)
+    describe('functional usage', () => {
+      beforeAll(async () => {
+        await setupProject()
+      }, 30000) // 30 second timeout as this might take a while
```

Comment:
this only needs to be called once per `describe` block, no need for an `beforeEach` since it takes quite a while to set up.